app-service-template/functions/sample.go

```diff
@@ -96,6 +97,61 @@ func (s *Sample) LogEventDetails(ctx interfaces.AppFunctionContext, data interfa
 	return true, event
 }
 
+// SendCommand is example of how to use the CommandClient to send commands to devices
+func (s *Sample) SendCommand(ctx interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
+	lc := ctx.LoggingClient()
+	lc.Debugf("SendCommand function called in pipeline '%s'", ctx.PipelineId())
+
+	if data == nil {
+		// Go here for details on Error Handle: https://docs.edgexfoundry.org/latest/microservices/application/ErrorHandling/
+		return false, fmt.Errorf("function SendCommand in pipeline '%s': No Data Received", ctx.PipelineId())
+	}
+
+	event, ok := data.(dtos.Event)
+	if !ok {
+		return false, fmt.Errorf("function SendCommand in pipeline '%s', type received is not an Event", ctx.PipelineId())
+	}
```

Comment:
maybe we don't need to limit the input has to be an event dto.
I can't imagine an use case to issue a Get command after receive the same event.
I suggest adding a custom config for this funciton to specify the device name and command name.

Reply:
@cloudxxx8 , It is just a sample to show how to use the Command Client. More importantly, for us to test the command client.